test/acceptance/resources/provisioned_backend_with_annotations.operator.manifest.yaml

```diff
@@ -39,4 +39,15 @@ spec:
     singular: provisionedbackend
     kind: ProvisionedBackend
     shortNames:
-      - pbk
\ No newline at end of file
+      - pbk
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: admin-to-provisionedbackends-stable
+  labels:
+    service.binding/controller: "true"
+rules:
+  - apiGroups: ["stable.example.com"]
+    resources: ["provisionedbackends"]
+    verbs: ["get", "list", "watch", "update", "patch", "delete"]
```

Comment:
do we really need `watch`, `update`, `patch` and `delete` permissions for tests to pass?

Reply:
I added these admin access due to CI failures, as these are only for CI only. let me re-run CI with reduced premission